### PR TITLE
Add static method in MonitorManager to get id given any input class

### PR DESCRIPTION
**Pull request by manish211**
@lafa pls review

### CODE REVIEW

File: jmx/src/main/java/com/lafaspot/jmetrics/common/MonitorManager.java [hunk 1]
```diff
@@ -53,9 +53,18 @@ public MonitorManager(final Class<T> clazz, final MonitorDirectory<T> monitorDir
         this.monitorDirectory = monitorDirectory;
         this.constNamespaceSet = constNamespaceSet;
         if (id == null) {
-            id = Integer.toString(clazz.getClassLoader().hashCode());
+            id = getId(clazz);
```
Comment (nvsnvikram) [inline jmx/src/main/java/com/lafaspot/jmetrics/common/MonitorManager.java]: getMonitorManagerId

File: jmx/src/main/java/com/lafaspot/jmetrics/common/MonitorManager.java [hunk 2]
```diff
@@ -53,9 +53,18 @@ public MonitorManager(final Class<T> clazz, final MonitorDirectory<T> monitorDir
...
         }
     }
+    /**
+     * Method to get monitor manager id derived from the class loader id of input class.
```
Comment (nvsnvikram) [inline jmx/src/main/java/com/lafaspot/jmetrics/common/MonitorManager.java]: Provides a unique id for MonitorManagers defined in class loader for web apps or jvm scope.

Comment (manish211) [reply jmx/src/main/java/com/lafaspot/jmetrics/common/MonitorManager.java]: Thanks. Done

File: jmx/src/main/java/com/lafaspot/jmetrics/common/MonitorManager.java (hunk 1 above)
Comment (manish211) [reply jmx/src/main/java/com/lafaspot/jmetrics/common/MonitorManager.java]: Done

**Pull request merged**